src/utils.ts

```diff
@@ -60,3 +60,25 @@ export const createComment = async (comment: string) => {
 
 export const generateCoverageSummary = async (jestCommand: string): Promise<string> =>
   await execCommand(jestCommand, summaryFormatter);
+
+const generateChangeSinceParam = () => {
+  const param = getInput('only-changes');
+
+  switch (param) {
+    case 'true':
+      return '--changeSince=context.payload.pull_request.base_ref';
```

Comment:
You need to check if is a pull_request action first.

```suggestion
      return `--changeSince=${context.payload.pull_request.base_ref}`;
```